source/modules/MEI.header.xml

```diff
@@ -1635,22 +1635,13 @@
     <desc xml:lang="en">Key captures information about tonal center and mode.</desc>
     <classes>
       <memberOf key="att.common"/>
-      <memberOf key="att.accidental"/>
       <memberOf key="att.bibl"/>
-      <memberOf key="att.pitch"/>
+      <memberOf key="att.keySig.anl"/>
       <memberOf key="model.workIdent"/>
     </classes>
     <content>
       <rng:text/>
     </content>
-    <attList>
-      <attDef ident="mode" usage="opt">
```

Comment:
Maybe I'm missing something, but without `@mode` how do you say something is in "F# minor"?

Reply:
It's part of `att.keySig.anl`.